pkg/utils/k8s.go

```diff
@@ -84,6 +96,49 @@ func ToK8sResourceRequirements(resources *core.Resources) (*v1.ResourceRequireme
 	return res, nil
 }
 
+func GetContainer(pod *v1.PodSpec, containerName string) (*v1.Container, error) {
+	// find flytepropeller container(s)
+	var containers []*v1.Container
+	for i := 0; i < len(pod.Containers); i++ {
+		if pod.Containers[i].Name == containerName {
```

Comment:
I don't think a Pod spec with multiple containers having the same name is valid. Unless the validation on PodTemplate is more lax?

Reply:
done. seems this was a remnant of checking the pod command instead of the name.